README.md

```diff
@@ -16,12 +16,14 @@
 
 <h3>👤 Hamid</h3>
 
-*GitHub: [@hamid-murambiwa](https://github.com/hamid-murambiwa)
-*Twitter: [@Hamid](https://twitter.com/Hamid87789454)
-*LinkedIn: [@hamid murambiwa](https://www.linkedin.com/in/hamid-murambiwa-8a9a9520a/)
+- GitHub: [@hamid-murambiwa](https://github.com/hamid-murambiwa)
+- Twitter: [@Hamid](https://twitter.com/Hamid87789454)
+- LinkedIn: [@hamid murambiwa](https://www.linkedin.com/in/hamid-murambiwa-8a9a9520a/)
 
-<h3>👤 @Willy Ntazama</h3>
+<h3>👤 Willy Ntazama</h3>
 
+- GitHub: [@ntzwilly](https://github.com/ntzwilly)
+- LinkedIn: [@ntazamawilly](https://linkedin.com/in/ntazama-willy-b676b7a
```

Comment:
- [ ] OPTIONAL
It will be a nice idea to keep your LinkedIn link's _URL_ implicit, like doing this in your markdown:
`LinkedIn: [@ntazamawilly](https://linkedin.com/in/ntazama-willy-b676b7a)`